catalogs/catalog-jdbc-doris/src/test/java/com/datastrato/gravitino/catalog/doris/integration/test/DorisTableOperationsIT.java

```diff
@@ -0,0 +1,313 @@
+import com.datastrato.gravitino.rel.indexes.Indexes;
+import com.datastrato.gravitino.rel.types.Type;
+import com.datastrato.gravitino.rel.types.Types;
+import com.datastrato.gravitino.utils.RandomNameUtils;
+import com.google.common.collect.Maps;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.testcontainers.shaded.org.awaitility.Awaitility;
+
+@Tag("gravitino-docker-it")
+public class DorisTableOperationsIT extends TestDorisAbstractIT {
+  private static final Type VARCHAR_255 = Types.VarCharType.of(255);
+  private static final Type VARCHAR_1024 = Types.VarCharType.of(1024);
+
+  private static Type INT = Types.IntegerType.get();
+
+  private static final String databaseName = GravitinoITUtils.genRandomName("doris_test_db");
+
+  @BeforeAll
+  public static void startup() {
+    TestDorisAbstractIT.startup();
+    createDatabase();
+  }
+
+  private static void createDatabase() {
+    DATABASE_OPERATIONS.create(databaseName, "test_comment", new HashMap<>());
+  }
+
+  private static Map<String, String> createProperties() {
+    Map<String, String> properties = Maps.newHashMap();
+    properties.put("replication_allocation", "tag.location.default: 1");
+    return properties;
+  }
+
+  @Test
+  public void testBasicTableOperation() {
+    String tableName = GravitinoITUtils.genRandomName("doris_basic_test_table");
+    String tableComment = "test_comment";
+    List<JdbcColumn> columns = new ArrayList<>();
+    JdbcColumn col_1 =
+        JdbcColumn.builder().withName("col_1").withType(INT).withComment("id").build();
+    columns.add(col_1);
+    JdbcColumn col_2 =
+        JdbcColumn.builder().withName("col_2").withType(VARCHAR_255).withComment("col_2").build();
+    columns.add(col_2);
+    JdbcColumn col_3 =
+        JdbcColumn.builder().withName("col_3").withType(VARCHAR_255).withComment("col_3").build();
+    columns.add(col_3);
+    Map<String, String> properties = new HashMap<>();
+
+    Distribution distribution = Distributions.hash(32, NamedReference.field("col_1"));
+    Index[] indexes = new Index[] {};
+
+    // create table
+    TABLE_OPERATIONS.create(
+        databaseName,
+        tableName,
+        columns.toArray(new JdbcColumn[0]),
+        tableComment,
+        createProperties(),
+        null,
+        distribution,
+        indexes);
+    List<String> listTables = TABLE_OPERATIONS.listTables(databaseName);
+    Assertions.assertTrue(listTables.contains(tableName));
+    JdbcTable load = TABLE_OPERATIONS.load(databaseName, tableName);
+    assertionsTableInfo(tableName, tableComment, columns, properties, indexes, load);
+  }
+
+  @Test
+  public void testAlterTable() {
```

Comment:
could you try to add more table change test like set properties, drop column, add index?

Reply:
test case is added